larpandoracontent/LArHelpers/LArFormattingHelper.h

```diff
@@ -0,0 +1,222 @@
+/**
+ *  @file   larpandoracontent/LArHelpers/LArFormatting.h
+ *
+ *  @brief  Header file for the formatting helper class
+ *
+ *  $Log: $
+ */
+#ifndef LAR_FORMATTING_HELPER_H
+#define LAR_FORMATTING_HELPER_H 1
+
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+namespace lar_content
+{
+
+/**
+ *  @brief  LArFormattingHelper class
+ */
+class LArFormattingHelper
+{
+public:
+
+    enum Colour : int;
+    enum Style : int;
```

Comment:
unsigned integers? Forward declaration for stylistic reasons? (would agree with this)

Reply:
Yep! they should be unsigned